exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,10 @@
 // Declare your function first
+let divide = function(a, b){
```

Comment:
Very good understanding of declaring and assigning but in this case you can simplify your code just by declaring the function strait away   function divde(){
}